canonical-web-and-design/certification.ubuntu.com
# API library

Here I've tried to create a CertificationAPI class which
tries to reflect in method and property names the shape of the
certification API.

This will hopefully make development easier and clearer down the line.

It also allowed me to simplify the existing view functions a fair bit.

QA
--

```
./run
```

Browse the site, check the bits that worked before still work.
